.gitignore

```diff
@@ -29,3 +29,9 @@ build/
 client/dist/reviews.js
 client/dist/reviews.js.LICENSE.txt
 
+server/dumps/*
+!server/dumps/.gitkeep
```

Comment:
This is cool- I've never seen the exclamation point used before!

Reply:
This was my first time using it. Very useful!